themes/edx.org/lms/templates/header.html

```diff
@@ -68,33 +68,33 @@ <h1 class="hd logo-header">
         </div>
       % endif
     </h1>
-    <button class="navbar-toggler navbar-toggler-right mt-2" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
This feature is not accessible. Focus should be trapped in the menu when it is opened.

Reply:
@HarryRein I'll need your help to see what we can do about this. This is just the built-in Bootstrap implementation.